elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -45,13 +73,32 @@ def render(element_html, data):
                 prefix = 'np.array('
                 suffix = ')'
 
+            # Mathematica reserved letters: C D E I K N O
+            mathematica_reserved = ['C', 'D', 'E', 'I', 'K', 'N', 'O']
+            if pl.inner_html(child) in mathematica_reserved:
+                mathematica_suffix = 'm'
+            else:
+                mathematica_suffix = ''
+
             # Create string for matlab and python format
-            matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=digits) + ';\n'
-            python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=digits) + suffix + '\n'
+            if pl.has_attrib(child, 'params-comment'):
+                matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=var_digits) + '; % ' + var_comment + '\n'
```

Comment:
I suggest using python f-strings to make this cleaner.

Reply:
Changed to f-strings